### DISCUSSION TITLE
Know if recorder is running

### DISCUSSION THREAD

**Discussion by floriangbh** [HaishinKit/HaishinKit.swift#1329] [Q&A]
Hello, 

Since https://github.com/shogo4405/HaishinKit.swift/pull/1319/files we are not unable to check if the `mixer.recorder.isRunning`. 

Is there any other way to check if the stream is running ? 

Use case : handling network cuts and different recording settings

Thanks

**Comment by shogo4405** [comment]
Certainly, it is necessary. I added the NetStream#isRecording property. https://github.com/shogo4405/HaishinKit.swift/commit/b223002a6d59c6ed9c4a4b40dd31dc4d38bc78bd

**Reply by floriangbh** [reply]
Thanks !
